helpdesk-maker-checker-service/src/main/java/org/symphonyoss/symphony/bots/helpdesk/makerchecker/model/check/AgentExternalCheck.java

```diff
@@ -178,15 +179,30 @@ public boolean isCheckerType(MakerCheckerMessage makerCheckerMessage) {
     return symApprovedMessages;
   }
 
+  @Override
+  public void removeAttachmentAndDirectory(SymAttachmentInfo symAttachmentInfo) {
+    File diretorio = new File("/tmp/" + symAttachmentInfo.getId());
+
+    File[] files = diretorio.listFiles();
+    if (files.length > 0) {
+      Arrays.stream(files)
+          .forEach(file -> file.delete());
+    }
+
+    diretorio.delete();
+  }
+
   private File getFileAttachment(SymAttachmentInfo symAttachmentInfo, SymMessage symMessage) {
-    File tempFile;
+    File diretorio = new File(TEMP_DIRECTORY + symAttachmentInfo.getId());
+    if (!diretorio.exists()) {
+      diretorio.mkdir();
+    }
+
+    File file = new File(diretorio + "/" + symAttachmentInfo.getName());
```

Comment:
Please use File.separator instead of '/' char.

Reply:
Done.